src/az390.java

```diff
@@ -2177,6 +2184,26 @@ else if (tz390.op_trace_type[index]==343)
                        {my_format="RRE";                                               // #500
                         my_operands="R1,R2";                                           // #500
                         }
+                    else if (tz390.op_trace_type[index]==141)                          // #612
+                       {if (tz390.op_name[index].equals("LOCGR"))                      // #612
+                           {my_format="RRF";                                           // #612
+                            my_operands="R1,R2,M3";                                    // #612
+                            }                                                          // #612
+                        else                                                           // #612
+                           {my_format="RRF";                                           // #612
+                            my_operands="R1,R2";                                       // #612
+                            }                                                          // #612
+                        }                                                              // #612
```

Comment:
Suggestion only. Lines 2189 and 2193 could be replaced by one line put either just before or just after the "if else".